docker/bitcoin-regtest/entrypoint.sh

```diff
@@ -19,11 +19,11 @@ cd /opt/coins/blockbook/bitcoin_regtest
 
 echo "Starting blockbook service"
 bin/blockbook \
-    -blockchaincfg=config/blockchaincfg.json \
+    -blockchaincfg=/opt/coins/blockbook/bitcoin_regtest/config/blockchaincfg.json \
     -datadir=/opt/coins/data/bitcoin_regtest/blockbook/db \
     -sync \
     -internal=:19021 \
     -public=:19121 \
-    -certfile=cert/blockbook \
     -explorer= \
-    -log_dir=logs
+    -log_dir=/opt/coins/blockbook/bitcoin_regtest/logs \
+    -dbcache=1073741824
```

Comment:
Is this needed?

Reply:
Absolutely not. This is ridiculous cache size for the regtest. I would use `-dbcache=0`

Reply:
sorry, i've copied id from somhere else :) 
531042e